doc/classes/EditorSettings.xml

```diff
@@ -234,12 +234,14 @@
 		</member>
 		<member name="editors/2d/bone_outline_size" type="int" setter="" getter="">
 			The outline size in the 2D skeleton editor (in pixels). See also [member editors/2d/bone_width].
+			[b]Note:[/b] Changes to this value only apply after modifying a [Bone2D] node in any way, or closing and reopening the scene.
 		</member>
 		<member name="editors/2d/bone_selected_color" type="Color" setter="" getter="">
 			The color to use for selected bones in the 2D skeleton editor. See also [member editors/2d/bone_outline_color].
 		</member>
 		<member name="editors/2d/bone_width" type="int" setter="" getter="">
```

Comment:
It seems off that the docs still think the property is an int. Is it set as an int somewhere else?

Reply:
I forgot to update `--doctool` – I've just done that.